### PR TITLE
No issue: create add_secrets to copy local secrets into correct files

### PR DESCRIPTION
**Pull request by MatthewTighe**
During attempts at debugging #25545, we have had some trouble with reproducing the issue regularly. One reason for that might be that builds from different people might include different subsets of features due to the lack of secrets included in the build process. 

It's an error-prone and time-consuming process to copy each secret into the appropriate place in our build scripts, so this attempts to remove that obstacle by making it easier to insert all our secrets during a local build.

I deleted the `.adjust_token` file because it appeared to only contain a placeholder, which was required in local release builds that were lacking the real secret. However, I was concerned that it would get overwritten when this script was used, and since it was already in the source tree it wouldn't be ignored correctly. This could lead to the secret being leaked pretty easily. Instead, the dummy file is created automatically during release builds where it is not present.

Excuse my amateurish python skills 😂 

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -318,6 +318,8 @@ android.applicationVariants.all { variant ->
             buildConfigField 'String', 'ADJUST_TOKEN', '"' + token + '"'
             println "(Added from .adjust_token file)"
         } catch (FileNotFoundException ignored) {
+            // Release builds will crash without a dummy file
+            new File("${rootDir}/.adjust_token").text = "--"
```
Comment (boek) [inline app/build.gradle:322]: If I recall correctly, crashing was intentional to make sure we weren't shipping a build without one?

File: tools/add_secrets.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+except IndexError:
+    print(
+        """
+        Expected 1 argument defined build variant. Probably 'nightly', 'beta', or 'release'.
+        For example: python3 tools/add_secrets.py nightly
+        """
+    )
+    sys.exit(2)
+
+# These mappings are borrowed from taskcluster/transforms/build.py and should be kept in sync.
```
Comment (boek) [inline tools/add_secrets.py:34]: I wonder if there is an easy way if we can share the mapping between the scripts so we don't have to manually keep them in sync 🤔

File: app/build.gradle (hunk 1 above)
Comment (MatthewTighe) [reply app/build.gradle:322]: I'll re-examine this PR once the repo migration will happen, but I believe this comment was about the actual build process and not the finished artifact. IIRC I had removed the empty `.adjust_token` file as part of this PR but we still wanted it present and empty during pipeline builds.

File: tools/add_secrets.py (hunk 2 above)
Comment (MatthewTighe) [reply tools/add_secrets.py:34]: I suppose a shared text file could be parsed between them. I'll consider if that's worth the effort when I come back to this PR

**Pull request closed without merging**